pkg/private/xtest/graphupdater/main.go

```diff
@@ -47,6 +50,16 @@ func writeLinksToFile() {
 	}
 }
 
+func writeIfIDsToFile() {
+	err := WriteIfIDsToFile(*topoFile, *ifIDsFile)
+	if err != nil {
+		fmt.Printf("Failed to write the ifIDs yaml file, err: %v\n", err)
+		os.Exit(1)
+	} else {
+		fmt.Printf("Successfully written the ifIDs yaml to %s\n", *ifIDsFile)
+	}
+}
```

Comment:
```suggestion
	err := WriteIfIDsToFile(*topoFile, *ifIDsFile)
	if err != nil {
		fmt.Printf("Failed to write the ifIDs yaml file, err: %v\n", err)
		os.Exit(1)
	}
	fmt.Printf("Successfully written the ifIDs yaml to %s\n", *ifIDsFile)
```

nit: the `else` block is unnecessary.

Reply:
Done.
<!-- Reviewable comment -OQqvHEV3oKnvPw6E4oK:b-896fix -->
<!-- Sent from Reviewable.io -->